src/components/imagegen_components/SynchronizedImageViewer.tsx

```
@@ -0,0 +1,85 @@
+
+    window.addEventListener(`sync-scroll-${syncGroup}`, handler);
+    return () => {
+      window.removeEventListener(`sync-scroll-${syncGroup}`, handler);
+    };
+  }, [syncGroup]);
+
+// At the top of the file, add the import
+import { useCallback } from 'react';
+
+// Add this helper function outside your component
+const debounce = (func: Function, wait: number) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)